tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -71,18 +81,69 @@ Ramdump Region Options:
 =========================================================================
 Please enter desired ramdump option as below:
         1 for ALL
-Please enter your input: 
+Please enter your input: 1
+
+ramdump_recv: No. of Regions to be dumped received
+
 ```
-5. Ramdump Tool receives the ram contents from target.
+6. Dump Tool receives the RAM contents from target.
 ```
-Target Handshake successful do_handshake
-Target entered to ramdump mode
 Receiving ramdump......
-ramdump_recv: ramdump_address = 02020000, ramdump_size = 968704
+
+=========================================================================
+```
+```
+Dumping data, Address: 0x02023800, Size: 968704bytes
+=========================================================================
 [===========================================================>]
-Ramdump received successfully
-copying ramdump_0x02020000_0x0210c800.bin to  $TIZENRT_BASEDIR/build/output/bin
+Ramdump received successfully..!
+```
+7. The dump tool will again provide a list of options for the user to chose from
+```
+Chose from the following options:-
+1. RAM Dump
+2. Userfs Dump
+3. Both Dumps
```

Comment:
we can change to "Both RAM dump and Userfs dump"

Reply:
Done